src/app/components/client/system-report-modal/system-report-form.vue

```diff
@@ -1,5 +1,24 @@
 <template>
 	<app-form name="systemReportForm">
+		<app-form-group name="issue_tracker_url" optional :label="$gettext('Issue Tracker Link')">
+			<p class="help-block">
+				<translate>
+					Paste the link to your ticket here. It's optional, but without it we can't get back to you
+					if we need more info to solve it.
+				</translate>
+				<span class="emoji emoji-huh" />
```

Comment:
Eh? No app-jolticon?

Reply:
It's a game jolt emoji, not a jolticon. iirc we don't have our emojis as jolticons.